Tools/cases_generator/opcode_metadata_generator.py

```diff
@@ -250,6 +250,10 @@ def generate_deopt_table(analysis: Analysis, out: CWriter) -> None:
         deopts.append((inst.name, deopt))
     for name, deopt in sorted(deopts):
         out.emit(f"[{name}] = {deopt},\n")
+    defined = set(analysis.opmap.values())
+    for i in range(256):
+        if i not in defined:
+            out.emit(f"[{i}] = {i},\n")
```

Comment:
Since we're not testing this at all in cpython, I'd suggest we at least add a couple of assertions to make sure this is correctly covering the range of opcodes:

```suggestion
    defined = set(analysis.opmap.values())
    for i in range(256):
        if i not in defined:
            deopts.append((f'{i}', f'{i}'))

    assert len(deopts) == 256
    assert len(set(x[0] for x in deopts)) == 256
    for name, deopt in sorted(deopts):
        out.emit(f"[{name}] = {deopt},\n")
```